spec/generators/pageflow/cancan/cancan_generator_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+# Generators are not automatically loaded by Rails
+require 'generators/pageflow/cancan/cancan_generator'
+
+RSpec.describe Pageflow::Generators::CancanGenerator, type: :generator do
+  # Tell the generator where to put its output (what it thinks of as Rails.root)
+  destination File.expand_path("../../../../../tmp", __FILE__)
+
+  before { prepare_destination }
```

Comment:
Can we move this and the `destination` call above to some shared context for `type: generator`. I don't think this setup code provides enough useful context to repeat it in all files.